### PR TITLE
Implement upload on frontend

### PR DESCRIPTION
**Pull request by tonyc856**

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Redirect } from "react-router-dom";
+
```
Comment (kramsey458) [inline src/App/App.js]: No white space between imports. SonarCloud doesn't like

File: src/Upload/Upload.js [hunk 2]
```diff
@@ -1,45 +1,75 @@
...
       files: []
    };
 
-   handleChange = event => {
-      this.setState({selectedFile: event.target.files[0]});
+   handleDrop = files => {
+      const newFiles = Array.from(files);
+      const filteredNewFiles = [...this.filterByFileSize(newFiles, MAX_FILE_SIZE)];
+      let currentFiles = this.state.files;
+
```
Comment (kramsey458) [inline src/Upload/Upload.js]: Empty line

File: src/Upload/Upload.unit.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   expect(instance.state.files).toEqual([]);
+});
+
+
+test('removeAllFiles', () => {
+   let files = [{size: 1000000}, {size: 5000}];
+   let expected = [];
+   const wrapper = shallow(<Upload />);
+   const instance = wrapper.instance();
+
```
Comment (kramsey458) [inline src/Upload/Upload.unit.test.js]: Remove empty line.

File: src/Upload/FilesList/FilesList.js [hunk 4]
```diff
@@ -2,16 +2,16 @@ import React, { Component } from 'react';
 import { Paper, Table, TableBody, TableHead, TableRow, TableCell } from '@material-ui/core';
 import FileItem from './FileItem'
 
-const labels = ["Filename", "Size", "Type", "Upload Progress"];
+const labels = ["Filename", "Size", "Type", ""];
```
Comment (kramsey458) [inline src/Upload/FilesList/FilesList.js]: Remove unnecessary extra member of the array ("").

File: src/Upload/FilesList/FileItem.js [hunk 5]
```diff
@@ -1,24 +1,49 @@
...
+   state = {
+      index: null,
+      name: "",
+      size: null,
+      type: ""
+   }
+
+   removeFile = () => {
+      this.props.onRemove(this.state.index);
+   }
```
Comment (brobinson15) [inline src/Upload/FilesList/FileItem.js]: Code by and large looks good, but we should discuss whether/where we're using semicolons (e.g. at the end of a declaration/instantiation as here, or at the end of a function block, as elsewhere).

Comment (tonyc856) [reply src/Upload/FilesList/FileItem.js]: I think we should just follow google's javascript style guide where we don't need to add semicolons at the end of a function declaration.

Google's semicolon style:

Semicolons should be included at the end of function expressions, but not at the end of function declarations. The distinction is best illustrated with an example:

var foo = function() {
  return true;
};  // semicolon here.

function foo() {
  return true;
}  // no semicolon here.

File: src/Upload/FilesList/FilesList.js (hunk 4 above)
Comment (tonyc856) [reply src/Upload/FilesList/FilesList.js]: Removed

**Pull request merged**